src/org/labkey/test/util/FileBrowserHelper.java

```diff
@@ -374,6 +374,33 @@ public void createFolder(String folderName)
         waitForElement(fileGridCell.withText(folderName));
     }
 
+    public void setDescription(String fileName, String description)
+    {
+        Window propWindow = editProperty(fileName);
+        setFormElement(Locator.name("Flag/Comment"), description);
+        propWindow.clickButton("Save", true);
+        _ext4Helper.waitForMaskToDisappear();
+    }
+
+    public String getFileDescription(String fileName)
+    {
+        List<String> fileList = getFileList();
+        int fileInd = fileList.indexOf(fileName);
+        if (fileInd > -1)
+        {
+            return getTexts(Locators.gridRow().childTag("td").position(7).findElements(getDriver())).get(fileInd);
+        }
+
+        return null;
+    }
```

Comment:
Does the `gridRow` locator not work for directories? Something like this would be more efficient:
```suggestion
    public String getFileDescription(String fileName)
    {
        return Locators.gridRow(fileName).childTag("td").position(7).findElement(getDriver()).getText();
    }
```
`WebElement.getText` is a relatively expensive operation and unnecessary calls should be avoided.  (`getTexts` and `getFileList` call it numerous times)